ons_alpha/bundles/views.py

```diff
@@ -0,0 +1,81 @@
+from typing import TYPE_CHECKING
+
+from django.core.exceptions import PermissionDenied
+from django.shortcuts import get_object_or_404, redirect
+from django.template.response import TemplateResponse
+from django.urls import reverse
+from django.utils.http import url_has_allowed_host_and_scheme
+from wagtail.admin import messages
+from wagtail.models import Page
+
+from .admin_forms import AddToBundleForm
+from .models import BundledPageMixin, BundlePage
+
+
+if TYPE_CHECKING:
+    from django.http import HttpRequest
+
+
+def add_to_bundle(request: "HttpRequest", page_to_add_id: "Page"):
+    page_to_add = get_object_or_404(Page, id=page_to_add_id)
+    page_to_add = page_to_add.specific
```

Comment:
Suggestion: This should work, and save a line
```suggestion
    page_to_add = get_object_or_404(Page.objects.specific(), id=page_to_add_id)
```